docs/uvbeam_tutorial.rst

```diff
@@ -150,7 +150,40 @@ b) Reading a CST E-field beam file
   >>> beam == beam3
   True
 
-c) Reading in the MWA full embedded element beam
+c) Reading a FEKO beam file (Power & E-field)
+**********************************
+.. code-block:: python
+
+  >>> import os
+  >>> from pyuvdata import UVBeam
+  >>> from pyuvdata.data import DATA_PATH
+  >>> import numpy as np
+
+  >>> filename_x = "OVRO_LWA_x.ffe"
+  >>> filename_y = "OVRO_LWA_y.ffe"
+  >>> feko_folder = "OVRO_LWA_FEKOBeams"
+  >>> feko_filename_x = os.path.join(DATA_PATH, feko_folder, filename_x)
+  >>> feko_filename_y = os.path.join(DATA_PATH, feko_folder, filename_y)
+
+  >>> pbeam_feko = UVBeam.from_file(
+  ...   feko_filename_x, beam_type='power', frequency=None,
```

Comment:
Why the `frequency=None`?

Reply:
oops, that's an error. The frequencies are defined in the file so do not need to be passed. Thanks for catching!